eng/pipelines/templates/steps/cosmos-emulator.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  EmulatorMsiUrl: "https://aka.ms/cosmosdb-emulator"
+  StartParameters: ''
```

Comment:
We should have start parameters like the above mentioned ones... 

"/DisableRateLimiting /PartitionCount=100 /Consistency=Strong"

Reply:
Just added them into the calling template:

```/NoExplorer /NoUI /DisableRateLimiting /PartitionCount=100 /Consistency=Strong```

Reply:
Read through the implementation of the module. The ```/NoExplorer``` option is on by default, and ```/DisableRateLimiting``` is also the default in the PowerShell cmdlet. These changes should do the trick. I'll verify it runs green and then merge.